quit/git.py

```diff
@@ -658,7 +658,8 @@ def commit(self, message, author_name, author_email, **kwargs):
         commiter = pygit2.Signature(commiter_name, commiter_email)
 
         # Sort index items
-        items = sorted(self.stash.items(), key=lambda x: (x[1][0], x[0]))
+        #items = sorted(self.stash.items(), key=lambda x: (x[1][0], x[0]))
+        items = list(self.stash.items())
```

Comment:
@huwenxin Is it unimportant to sort the index? Could we maybe even just use `self.stash.items()` instead of `items` and use `for item in items` below?

Reply:
During testing I found that sometimes the self.stash.items() was None so that sorted would result in error. It's also possible to use self.stash.items() instead of items below. By the way, I will list the changes/features I've made and send you later, as well as a newer version of code.